### PR TITLE
Remove GCP arch viz since it's moved to Forseti

### PR DESCRIPTION
**Pull request by ryanmcdowell**
The person who maintains GCP arch viz is no longer able to maintain it and the tool has been pulled into Forseti.

https://cloud.google.com/blog/products/identity-security/understand-gcp-organization-resource-hierarchies-with-forseti-visualizer

### CODE REVIEW

File: tools/gcp-arch-viz/README.md [hunk 1]
```diff
@@ -1,42 +0,0 @@
-# GCP Architecture Visualizer
```
Comment (jaketf) [inline tools/gcp-arch-viz/README.md:1]: IMO it would be good to leave a README here that points to the tool's new home.

Comment (ryanmcdowell) [reply tools/gcp-arch-viz/README.md:1]: Would removing the sub-directory but modifying the root README to point to the following work or do we always want to keep the sub-directory?

https://github.com/forseti-security/forseti-visualizer

Comment (jaketf) [reply tools/gcp-arch-viz/README.md:1]: that makes sense just put link in root readme.

Comment (ryanmcdowell) [reply tools/gcp-arch-viz/README.md:1]: Updated

**Pull request merged**